Makefile

```diff
@@ -68,7 +68,7 @@ docker-push:
 
 setup-appmesh-sdk-override:
 	@if [ "$(APPMESH_SDK_OVERRIDE)" = "y" ] ; then \
-	    ./appmesh_models_override/setup.sh ; \
+	    APPMESH_PREVIEW=1 ./appmesh_models_override/setup.sh ; \
```

Comment:
there is a PR to make it `APPMESH_PREVIEW=y`: https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/298/files
maybe we should merge that change into preview branch as well, and rebase to avoid this two branch diverge too far..

Reply:
all other looks good to me

Reply:
yikes! a little too late. will sync